src/psyclone/dynamo0p3_builtins.py

```diff
@@ -508,10 +533,11 @@ def gen_code(self, parent):
         :type parent: :py:class:`psyclone.f2pygen.BaseGen`
 
         '''
-        # We multiply one element of field f1 (2nd arg) by the first scalar
-        # (1st arg), add it to the product of the corresponding element of
-        # a second field (4th arg) with the second scalar (4rd arg) and
-        # write the value back into the element of field f1.
+        # We multiply one element of field f1 (2nd arg) by the first real
```

Comment:
Commas please. (i.e. "first, real, scalar" as opposed to the first scalar that is real).

Reply:
Done.